internal/pkg/aws/apprunner/apprunner.go

```diff
@@ -51,6 +54,8 @@ type AppRunner struct {
 
 // New returns a Service configured against the input session.
 func New(s *session.Session) *AppRunner {
+	s.Config.Region = aws.String("us-east-1")
+	s.Config.Endpoint = aws.String("https://fusion.gamma.us-east-1.bullet.aws.dev/")
```

Comment:
Does not feel super right exposing these endpoints : ( . What do you think of `cherry-pick` or `rebase` locally when you run the tests?